misc/testerina/modules/testerina-core/src/test/resources/coverage/project/examples_module2/source2.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/io;
+
+public function main(string... args) {
+
+    boolean[] booleanValues = [true, false];
+
+    foreach boolVal in booleanValues {
+        sourceFunc2(boolVal);
+    }
+
+    io:println("Hello, World2! 1");
+}
+
+function sourceFunc2(boolean boolVal) {
+    io:println("Hello, World2! 2");
+
+    if (boolVal) {
+        io:println("Hello, World2! 2 if block boolVal: " + boolVal);
+    } else {
+        io:println("Hello, World2! 2 else block boolVal: " + boolVal);
+    }
+}
```

Comment:
Shall we add a new line at the end of the file?

Reply:
done